README.md

```diff
@@ -17,28 +17,34 @@ There's a file named `config.js` at the base directory of the code samples. You
 your authentication token.
 
 ```
-[-] .
- |  config.js
- | [-] call-recordings
- |  |  01-get-all-call-recordings.js
- |  |  02-get-last-two-weeks-call-recordings.js
- |  |  03-get-one-call-recording.js
- |  |  04-download-all-call-recordings.js
- |  |  05-download-last-two-weeks-call-recordings.js
- |  `  06-download-one-call-recording.js
- | [-] users
- |  |  01-get-all-users.js
- |  `  02-get-users-with-call-recordings-from-yesterday.js
- | [-] locations
- |  |  01-get-locations-optionally-by-country.js
- |  `  02-find-the-location-with-more-users.js
- | [-] departments
- |  |  01-list-your-organization-departments.js
- |  `  02-find-the-department-with-more-users.js
- | [-] organizations
- |  |  01-get-all-organizations.js
- |  `  02-get-organization-by-code.js
- `  README.md
+.
```

Comment:
Maybe we should just remove this entire heirarchy.  Not sure it adds much and it will always get out of date.

Reply:
Made more changes to README.md because of outdated text